src/modules/clusters/cluster-list/cluster-list.js

```diff
@@ -364,6 +361,22 @@
                 cluster.isProfilingEnabled === "Unknown";
         }
 
+        function getTemplate(cluster) {
+            var template = "";
+
+            if (cluster.jobType === "UnmanageCluster" && cluster.currentStatus === "failed") {
```

Comment:
can we add  cluster.currentStatus === "failed" this condition on top as it's common for all.

Reply:
@cloudbehl we can't add this condition at the top as the 4th `if else condition` checks for `cluster.currentStatus !== "failed" `